Core.BackgroundJobs/Model/ReadModels/ScheduleDataProcessingRegistrationReadModelUpdates.cs

```diff
@@ -29,20 +27,23 @@ public class ScheduleDataProcessingRegistrationReadModelUpdates : IAsyncBackgrou
             IPendingReadModelUpdateRepository updateRepository,
             IDataProcessingRegistrationReadModelRepository readModelRepository,
             IDataProcessingRegistrationRepository dataProcessingRegistrationRepository,
-            IItContractRepository contractRepository,
             ITransactionManager transactionManager)
         {
             _updateRepository = updateRepository;
             _readModelRepository = readModelRepository;
             _dataProcessingRegistrationRepository = dataProcessingRegistrationRepository;
-            _contractRepository = contractRepository;
             _transactionManager = transactionManager;
         }
 
         public Task<Result<string, OperationError>> ExecuteAsync(CancellationToken token = default)
         {
             var updatesExecuted = 0;
-            var alreadyScheduledIds = new HashSet<int>();
+            var idsOfDprsAlreadyInQueueForUpdate = _updateRepository
```

Comment:
vi cacher lige alle dem der allerede er i kø som udgangspunkt for vores filtrering